CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.0
+  * Update key properties for streams, fixes issue [#34](https://github.com/singer-io/tap-pendo/issues/34) with missing records [#88](https://github.com/singer-io/tap-pendo/pull/88)
```

Comment:
AFAIK we have made primary key changes to event streams only should specifically mention the event streams here.

Reply:
The PR is self descriptive of the change, so this level of detail looks appropriate and acceptable in the changelog.